tools/bin/ecflow/processes/main/models/prep_and_run_vic_twice.py

```diff
@@ -0,0 +1,119 @@
+
+    forcing_prefix = os.path.join(config_dict[section]['Subd_Out_Dir'],
+                                  'forcing_{0}-{1}.'.format(
+                                      vic_start_date.replace('-', ''),
+                                      vic_end_date.replace('-', '')))
+
+    # parse out year, month, and day from the model dates, which have
+    # the form YYYY-MM-DD
+    start = parse(vic_start_date)
+    end = parse(vic_end_date)
+    save_state = parse(config_dict[section]['vic_save_state'])
+
+    # generate the path to the initial state file
+    in_state = os.path.join(config_dict[section]['InStatePath'],
+                            'state.%s%s%s_00000.nc' % (
+                                vic_start_date[:4], vic_start_date[5:7],
+                                vic_start_date[8:10]))
+
+    # run to the date at which we first want to save the state (for tomorrow's
+    # monitor)
+    kwargs = {
+        'Start_Year': start.year,
+        'Start_Month': start.month,
+        'Start_Day': start.day,
+        'End_Year': save_state.year,
+        'End_Month': save_state.month,
+        'End_Day': save_state.day,
+        'State_Year': save_state.year,
+        'State_Month': save_state.month,
+        'State_Day': save_state.day,
+        'In_State': in_state,
+        'State_Path': config_dict[section]['StatePath'],
+        'Result_Path': config_dict[section]['OutputDirRoot'],
+        'Forcing_Prefix': forcing_prefix}
+
+    model_tools.replace_var_pythonic_config(
+        global_template, global_file_path, header=None, **kwargs)
+
+    # mpirun -np 16 vic_image.exe -g global_file
+    subprocess.run([config_dict['ECFLOW']['MPIExec'], '-np',
+                    str(config_dict['ECFLOW']['Cores']),
+                    config_dict['ECFLOW']['Executable'], '-g',
+                    global_file_path])
+
+    # generate the path to the initial state file
+    in_state = os.path.join(config_dict[section]['StatePath'],
+                            'state.{0}{1:02d}{2:02d}_00000.nc'.format(
+                                save_state.year, save_state.month,
+                                save_state.day))
+    state_date = end + timedelta(days=1)
+    # run to current date & save state to initialize medium-range forecast
+    kwargs = {
+        'Start_Year': save_state.year,
+        'Start_Month': save_state.month,
+        'Start_Day': save_state.day,
+        'End_Year': end.year,
+        'End_Month': end.month,
+        'End_Day': end.day,
+        'State_Year': state_date.year,
+        'State_Month': state_date.month,
+        'State_Day': state_date.day,
+        'In_State': in_state,
+        'State_Path': config_dict[section]['StatePath'],
+        'Result_Path': config_dict[section]['OutputDirRoot'],
+        'Forcing_Prefix': forcing_prefix}
+
+    model_tools.replace_var_pythonic_config(
+        global_template, global_file_path, header=None, **kwargs)
+
+    # mpirun -np 16 vic_image.exe -g global_file
```

Comment:
or maybe just convert it into a comment like `the line below executes the following command:`